### ISSUE TITLE
Hypobilirubinemia

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5945]
**Preferred term label:** Hypobilirubinemia

**Synonyms**

**Definition (free text, please give PubMed ID)** A decreased amount of bilirubin in the blood., PMID:21807875

**Parent term (use hpo.jax.org/app)** Abnormal circulating metabolite concentration HP:0032180

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
